### PR TITLE
HA: Set Optional ordering for Ceilometer startup

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/ceilometer/recipes/server_ha.rb [hunk 1]
```diff
@@ -60,9 +60,17 @@
 if node[:ceilometer][:use_mongodb]
   pacemaker_order "o-ceilometer-mongo" do
     score "Mandatory"
```
Comment (vuntz) [inline chef/cookbooks/ceilometer/recipes/server_ha.rb]: I'm unsure if we want to keep it Mandatory. That sounds wrong given the fact that there are many daemons in the group that might still just work fine if mongodb goes down.

File: chef/cookbooks/ceilometer/recipes/server_ha.rb [hunk 2]
```diff
@@ -60,9 +60,17 @@
...
-    ordering "cl-mongodb cl-g-ceilometer-server"
+    ordering "cl-mongodb cl-#{group_name}"
+    action :create
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 end
 
+crowbar_pacemaker_order_only_existing "o-cl-#{group_name}" do
+  ordering [ "rabbitmq", "cl-keystone", "cl-#{group_name}" ]
```
Comment (vuntz) [inline chef/cookbooks/ceilometer/recipes/server_ha.rb]: Haven't put postgresql as I'm unsure if it should be optional or mandatory (like for mongodb). Any opinion,

Comment (dirkmueller) [inline chef/cookbooks/ceilometer/recipes/server_ha.rb]: not sure if it matters, we have the dependency on cl-keystone already.

**Comment by dirkmueller** [comment]
+1

**Comment by vuntz** [comment]
Added a commit to also depend on postgresql.

**Comment by aspiers** [comment]
:+1:

**Pull request merged**